### PR TITLE
[wallet-ext] - add Banxa as onramp provider

### PR DESCRIPTION
**Pull request by mamos-mysten**
## Description 

- Adds Banxa as an onramp provider. 
- Updates the UI to match the latest spec.

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: apps/wallet/src/manifest/manifest.json [hunk 1]
```diff
@@ -30,8 +30,8 @@
 	],
 	"web_accessible_resources": [
 		{
-			"resources": ["dapp-interface.js"],
-			"matches": ["<all_urls>"]
+			"resources": ["ui.html"],
+			"matches": ["https://*.banxa.com/*"]
```
Comment (mamos-mysten) [inline apps/wallet/src/manifest/manifest.json:34]: this allows Banxa to redirect back to the wallet

File: apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx [hunk 2]
```diff
@@ -408,23 +407,21 @@ function TokenDetails({ coinType }: TokenDetailsProps) {
...
+									{!accountHasSui && (
 										<div className="flex flex-col gap-5">
 											<div className="flex flex-col flex-nowrap justify-center items-center text-center px-2.5">
-												<SvgSuiTokensStack className="h-14 w-14 text-steel" />
-												<div className="flex flex-col gap-2 justify-center">
-													<Text variant="pBodySmall" color="gray-80" weight="normal">
-														To send transactions on the Sui network, you need SUI in your wallet.
-													</Text>
-												</div>
+												<Text variant="pBodySmall" color="gray-80" weight="normal">
```
Comment (pchrysochoidis) [inline apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx:417]: Is it intentional to show this on all networks? Only mainnet should be better I think and keep the previous wording for everything except mainnet.

File: apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx [hunk 3]
```diff
@@ -434,13 +431,17 @@ function TokenDetails({ coinType }: TokenDetailsProps) {
 									) : null}
 									<div className="grid grid-cols-3 gap-3 w-full">
 										<LargeButton
+											spacing="sm"
```
Comment (pchrysochoidis) [inline apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx:434]: Now that this is a bigger button it doesn't look right especially with the faucet button there as well for networks other than mainnet

File: apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx (hunk 2 above)
Comment (mamos-mysten) [reply apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx:418]: This was not intentional, should be fixed up now!

**Pull request merged**